src/index.ts

```diff
@@ -282,12 +282,25 @@ export async function build({
 		...(await glob('.deno/**/*', workPath)),
 	};
 
-	console.log('Detected source files:');
-
 	if (denoTsConfig) {
 		sourceFiles.add(denoTsConfig);
 	}
 
+	if(config.includeFiles) {
```

Comment:
Can we move this block to _after_ the source files one?

Reply:
After which block exactly?

Reply:
The "Detected source files" one